### ISSUE TITLE
Redesign the login page

### ISSUE THREAD

**Issue by andersonjeccel** [mautic/mautic#13167]
### Description
The current login page lacks an appealing and user-friendly design, but it's the first screen users see everyday; we aim to redesign the login page for improved aesthetics and usability.

### Impact
Redesigning the login page will make a positive first impression, increase user engagement, and provide a more intuitive login experience, potentially leading to higher user retention and satisfaction.

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by andersonjeccel** [comment]
Low priority, I won't tackle it.
